roles/ispmail-rspamd/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Add rspamd APT signing key
+  apt_key:
+    url: https://rspamd.com/apt-stable/gpg.key
+    state: present
+
+- name: Add Rspamd APT repository
+  apt_repository:
+    repo: >-2
+      {{ item }} [arch=amd64]
+      http://rspamd.com/apt-stable/
+      {{ ansible_distribution_release }} main
+    state: present
+    filename: rspamd
+  notify: update apt cache
+  loop:
+    - deb
+    - deb-src
+
+- name: Force trigger apt cache update if needed
+  meta: flush_handlers
+
+- name: Install rspamd packages
+  apt:
+    name:
```

Comment:
As long as there is only one package to install, we can simplify this to:
```yaml
    name: rspamd
```

Reply:
done